packages/venia-ui/lib/components/MiniCart/ProductList/item.css

```diff
@@ -48,6 +54,7 @@
 }
 
 .editButton {
-    grid-column: 3 / span 1;
+    grid-column: 2 / span 1;
     grid-row: 1 / span 1;
+    min-width: 1rem;
```

Comment:
I added this because when the delete icon is clicked, we remove the button because of which the second column in the grid becomes empty and the grid realigns. It looks like things are moving around. To avoid it, I have added a min-width but still, it moves around. Any help is appreciated.